vars/folioHelm.groovy

```diff
@@ -176,6 +176,11 @@ String generateModuleValues(RancherNamespace ns, String moduleName, String modul
         moduleConfig << [consortiumEnabled: "true"]
     }
 
+    //Enable cross tenant extra env
+    if (Constants.CROSS_TENANT_ENABLED.contains(moduleName) && ns.enableConsortia){
```

Comment:
As we already have couple of customizations for enableConsortia maybe reasonable to create an if statement for enableConsortia and a couple sub-conditions